cypress/e2e/hub/approvals.cy.ts

```diff
@@ -1,12 +1,46 @@
 import { Approvals } from './constants';
+import { randomString } from '../../../framework/utils/random-string';
 
 describe('Approvals', () => {
+  let thisCollectionName: string;
+  let namespace: string;
+
   before(() => {
     cy.hubLogin();
+    cy.getNamespace('ibm');
+    cy.navigateTo('hub', Approvals.url);
+  });
+
+  after(() => {
+    cy.deleteCollection(thisCollectionName, 'ibm', 'published');
   });
 
   it('it should render the approvals page', () => {
+    cy.verifyPageTitle(Approvals.title);
+  });
+
+  it('user can upload a new collection and approve it', () => {
+    thisCollectionName = 'test' + randomString(4).toLowerCase();
+    namespace = 'ibm';
```

Comment:
Is ibm standard namespace in the system, what if its deleted?
I rather suggest creating own namespace with random string, but its up to discussion. The solution with ibm namespace will be faster, but it can cause potential troubles. For example when two tests will run over cleaned database, both will query namespaces, both will at the same time recieve that ibm not exist, both will attempt to create it, both will fail.